src/components/Header/Header.module.scss

```diff
@@ -1,28 +1,48 @@
 @import '@stihl-design-system/components/styles/scss';
 
+// **************************************************************
+// The STIHL Design System Grid mixin is used in this example app.
+// It is based on the native CSS Grid and provides a simple way to create
+// responsive grid layouts.
+// The STIHL Design System Grid is built to be used in a standard "row-based" layout
+// where every row is a grid container and every column is a grid item.
+// This allows developers to create complex, responsive layouts while maintaining
+// a clear and modular structure.
+// This example app demonstrates the usage of the STIHL Design System Grid.
+// See full documentation for the STIHL Design System Grid at:
+// https://main--63440bbb95889041542a5ba3.chromatic.com/?path=/docs/styles-grid-grid--documentation
+// **************************************************************
 .top-bar {
-  @include ds-grid-full-column;
+  // Initializes the grid layout for the top bar container.
+  // This enables us to place the contents inside the top bar along
+  // the predefined grid areas.
   @include ds-grid;
+
+  // DO WE NEED THIS? width is already set to 100vw ?
+  @include ds-grid-full-column;
+  // And this?! width is automatically set to 100% ?
   width: 100vw;
```

Comment:
I'd remove width and keep the full-column, just to be "explicit" about what column the top-bar is located in. Makes it easier to identify its layout properties.

Reply:
I understand. But it does not have any effect, right? For it to have an effect the parent element would need to be a ds-grid, too?

Reply:
Yeah right, the parent has display:block on it. Then we can remove it :)